toolkit/tests/pastaporto/test_tokens.py

```diff
@@ -72,3 +75,19 @@ async def _():
         decode_service_to_service_token(
             test_token, "secret", issuer="test", audience="association-backend"
         )
+
+
+@test("generate a token")
+async def _():
+    token = generate_service_to_service_token("secret", issuer="me", audience="you")
+
+    assert token
+
+
+@test("raise ValueError if secret is empty")
```

Comment:
this file contains multiple tests for different types of tokens, maybe the description should be more clear?

```suggestion
@test("Secret is required when creating a service-to-service token")
```